### ISSUE TITLE
Helm chart issue - error calling gt: incompatible types for comparison

### ISSUE THREAD

**Issue by zaggash** [rancher/rancher#25312]
Easily reproducible by following the Rancher doc.

$ helm version
version.BuildInfo{Version:"v3.0.3", GitCommit:"ac925eb7279f4a6955df663a0128044a8a6b7593", GitTreeState:"clean", GoVersion:"go1.13.6"}

helm install rancher rancher-stable/rancher \
  --namespace cattle-system --set auditLog.level=1  \   
  --set hostname=myhostname.name.com \
  --set ingress.tls.source=letsEncrypt \
  --set letsEncrypt.email=test@example.com \
--version v2.3.4
Error: template: rancher/templates/deployment.yaml:81:7: executing "rancher/templates/deployment.yaml" at <gt .Values.auditLog.level 0.0>: error calling gt: incompatible types for comparison

Removing the --set auditLog.level=1  works.

**Comment by zaggash** [comment]
https://github.com/rancher/rancher/issues/23516

**Comment by tmbs** [comment]
Same issue with Helm v3.1.1 and Rancher v2.3.5

**Comment by rawmind0** [comment]
PRs #25628 and #25626 should address the issue

**Comment by deniseschannon** [comment]
Available to test in v2.3.6-rc3

**Comment by aaronRancher** [comment]
The bug is reproduced with Rancher server `rancher\rancher:v2.3.5`

Installed with:
```
helm install rancher rancher-latest/rancher \
   --namespace cattle-system \
   --set auditLog.level=1 \
   --set hostname=<hostname> \
   --set ingress.tls.source=letsEncrypt \
   --set letsEncrypt.email=<email> --version v2.3.5
```
Result:
```
Error: template: rancher/templates/deployment.yaml:81:7: executing "rancher/templates/deployment.yaml" at <gt .Values.auditLog.level 0.0>: error calling gt: incompatible types for comparison
```

The bug fix is verified with Rancher server `rancher\rancher:v2.3.6-rc3`

Installed with:
```
helm install rancher rancher-latest/rancher \
  --namespace cattle-system \
  --set auditLog.level=1 \
  --set hostname=<hostname> \
  --set ingress.tls.source=letsEncrypt \
  --set letsEncrypt.email=<email> --version v2.3.6-rc3
```
Installation is successful and UI can be accessed.
